### ISSUE TITLE
Instead of information about Union and Region it says Unknown

### ISSUE THREAD

**Issue by AlexanderSanya** [xAkre/Wuthering-Waves-RPC#16]
I did everything according to the instructions, as you indicated. After starting the game and loading it into my world, “Unknown” appears on my account in the “Union” and “Region” fields... I will attach the log file. How can I solve this problem? I did as stated in message #1 (“RPC - Everything show as “unknown”), but it didn’t help me. The error in the log itself is not the same.
[log.txt](https://github.com/user-attachments/files/17285882/log.txt)
P.S. I completely deleted the "LocalStorage" folder, but it didn't help me: the next time I started the game it created a new folder and 2 new files, but the result in the "log" file was still the same

**Comment by xAkre** [comment]
Sorry for the very late reply. I will have a look at this today.

**Comment by xAkre** [comment]
I cannot seem to reproduce. I reinstalled WuWa and the RPC too and it seems to work. Are you sure you set the correct kuro games uid during setup?

**Comment by AlexanderSanya** [comment]
Of course, I wrote it manually, and pasted it, copied it in the game, but the problem was not solved. I also tried installing the installer several times, and also cloned the repository, tried it many times - it didn’t help...

**Comment by xAkre** [comment]
I would need to know what your LocalStorage files look like. Would you mind going through these steps?:

- Install [Sqlite](https://www.sqlite.org/download.html)
- Go into the folder {Your Game Directory}/Wuthering Waves Game/Client/Saved/LocalStorage via the command line
- For every file there run sqlite3 {The File Name}
- Once in the sqlite command line run `SELECT * FROM LocalStorage WHERE key = 'SdkLevelData'`
- Check if the result follows a format like this;
```
{
    "___MetaType___":"___Map___",
    "Content": [
        ["535414272", [
           {
                "Region": "Europe",
                "Level": 4
            }
        ]],
        ["536678859", [
            {
                "Region": "Europe",
                "Level": 3
            }
        ]],
        ["536789175", [
            {
                "Region": "Europe",
                "Level": 22
            }]
        ]
    ]
}
```

While you are at it, make sure the kuro games uid you pass during setup is somewhere in those results.

**Comment by xAkre** [comment]
I am closing this for now due to inactivity
